onnx/reference/ops/aionnx_preview_training/_op_list.py

```diff
@@ -5,6 +5,7 @@
 
 import textwrap
 from typing import Any, Dict, Union
+from typing import Optional as TOptional
```

Comment:
Why is this alias needed?

Reply:
If we use Optional directly from typing here, there will be some errors:
```
FAILED onnx/test/reference_evaluator_backend_test.py::TestOnnxBackEndWithReferenceEvaluator::test_adagrad - AssertionError: Unable to run test 'test_adagrad' due to Unexpected variable type typing.Optional and class_name='Optional'.
FAILED onnx/test/reference_evaluator_backend_test.py::TestOnnxBackEndWithReferenceEvaluator::test_adagrad_multiple - AssertionError: Unable to run test 'test_adagrad_multiple' due to Unexpected variable type typing.Optional and class_name='Optional'.
FAILED onnx/test/reference_evaluator_backend_test.py::TestOnnxBackEndWithReferenceEvaluator::test_adam - AssertionError: Unable to run test 'test_adam' due to Unexpected variable type typing.Optional and class_name='Optional'.
FAILED onnx/test/reference_evaluator_backend_test.py::TestOnnxBackEndWithReferenceEvaluator::test_adam_multiple - AssertionError: Unable to run test 'test_adam_multiple' due to Unexpected variable type typing.Optional and class_name='Optional'.
FAILED onnx/test/reference_evaluator_backend_test.py::TestOnnxBackEndWithReferenceEvaluator::test_momentum - AssertionError: Unable to run test 'test_momentum' due to Unexpected variable type typing.Optional and class_name='Optional'.
FAILED onnx/test/reference_evaluator_backend_test.py::TestOnnxBackEndWithReferenceEvaluator::test_momentum_multiple - AssertionError: Unable to run test 'test_momentum_multiple' due to Unexpected variable type typing.Optional and class_name='Optional'.
FAILED onnx/test/reference_evaluator_backend_test.py::TestOnnxBackEndWithReferenceEvaluator::test_nesterov_momentum - AssertionError: Unable to run test 'test_nesterov_momentum' due to Unexpected variable type typing.Optional and class_name='Optional'.
```
I guess in some reference code they use Optional from typing as TOptional to distinguish Optional from `onnx.reference.ops.op_optional`.